Doc/library/sqlite3.rst

```diff
@@ -624,9 +617,10 @@ Cursor Objects
 
    .. method:: executemany(sql, seq_of_parameters)
 
-      Executes an SQL command against all parameter sequences or mappings found in
-      the sequence *seq_of_parameters*.  The :mod:`sqlite3` module also allows
-      using an :term:`iterator` yielding parameters instead of a sequence.
+      Executes a :ref:`parameterized <sqlite3-placeholders>` SQL command
+      against all parameter sequences or mappings found in the sequence
+      *seq_of_parameters*. The :mod:`sqlite3` module also allows using an
+      :term:`iterator` yielding parameters instead of a sequence.
```

Comment:
I wonder if it makes sense to add an example to `Doc/includes/sqlite3/execute_1.py` for this case.

Reply:
There is already an iterator example (standalone, just above the placeholder mark). That should be sufficient, I guess.